01-Intro.Rmd

```diff
@@ -17,13 +17,13 @@ While some of the instructions in this book are general guidelines/ best practic
 
 Someone unfamiliar with your project should be able to look at your R code, data files, and outputs and understand in detail what you did and why. This *someone* will very likely be your future self! We very often discover flaws in our initial preparation of the data, get access to new data, or decide that the parameterization of a particular model was not correct. This means that the analysis you did last week/ month/ year will probably need to be redone. If you have organized and documented your work clearly, then repeating the analysis will be much, much easier.
 
-Staff turnover is also a reality in all organizations. If you leave Birds Canada for greener pastures, your colleague/ replacements will also need to know where to find your scripts, files, and outputs. This will better ensure knowledge is not lost and time is not wasted recreating your workflows.
```

Comment:
I also would remove the space after '/', I think it should be "colleagues/replacements", (and the same throughout).